README.md

```diff
@@ -519,6 +519,11 @@ The `pointcloud_postgis` extension adds functions that allow you to use PostgreS
 > 
 >     POINT Z (-127 45 124)
 
+**PC_Envelope(pcpatch)** returns **geometry**<br/>
```

Comment:
Let's make the doc more verbose, as done for other functions. We can reuse the "Useful for performing intersection tests with geometries" statement I guess.